File: src/main/java/me/nobaboy/nobaaddons/features/chatcommands/impl/party/CancelCommand.java
package me.nobaboy.nobaaddons.features.chatcommands.impl.party;

import me.nobaboy.nobaaddons.NobaAddons;
import me.nobaboy.nobaaddons.features.chatcommands.ChatContext;
import me.nobaboy.nobaaddons.features.chatcommands.IChatCommand;

public class CancelCommand implements IChatCommand {
    @Override
    public String name() {
        return "cancel";
    }

    @Override
    public void run(ChatContext ctx) {
        WarpCommand.cancel = true;
    }

    @Override
    public boolean isEnabled() {
        return NobaAddons.config.partyWarpCommand;
    }
}

File: src/main/kotlin/me/nobaboy/nobaaddons/features/chatcommands/impl/party/CancelCommand.kt
package me.nobaboy.nobaaddons.features.chatcommands.impl.party

import me.nobaboy.nobaaddons.NobaAddons
import me.nobaboy.nobaaddons.features.chatcommands.ChatContext
import me.nobaboy.nobaaddons.features.chatcommands.IChatCommand

class CancelCommand : IChatCommand {
    override val name: String = "cancel"

    override val isEnabled: Boolean
        get() = NobaAddons.config.partyWarpCommand

    override val bypassCooldown: Boolean = true

    override fun run(ctx: ChatContext) {
        WarpCommand.cancel = true
    }
}